integrations/fathom/src/index.ts

```diff
@@ -26,7 +27,16 @@ export const handleFetchEvent: FetchPublishScriptEventCallback = async (
         return;
     }
 
-    return new Response(script.replace('<TO_REPLACE>', siteId), {
+    const trackExternalLinks =
+        environment.siteInstallation?.configuration?.track_external_links ??
+        environment.spaceInstallation?.configuration?.track_external_links ??
```

Comment:
You can omit `environment.spaceInstallation...` from here as well

Reply:
Done